ansible/group_vars/catalog-next/vars.yml

```diff
@@ -0,0 +1,32 @@
+---
```

Comment:
note to self: add a comment about inventory catalog-next vars won't override these.